packs/equipment/fury-cocktail-greater.json

```diff
@@ -11,7 +11,7 @@
+            "value": "<p>@UUID[Compendium.pf2e.equipment-effects.Item.Effect: Fury Cocktail (Greater)]{Effect: Fury Cocktail (Greater)}</p>\n<p><strong>Activate</strong> <span class=\"action-glyph\">1</span> Interact</p>\n<p>A fury cocktail is a fortifying ginger beer spiked with rum and a mixer. It's rumored to have originated from a barbarian-themed festival in a popular mead hall specializing in alchemical beverages.</p>\n<p><strong>Benefit</strong> You gain a +4 item bonus to melee attack rolls and an additional effect depending on the additive chosen when the brew is created. The effects last 1 hour.</p>\n<ul>\n<li><strong>Animalistic</strong> Lemon juice and powdered claws or talons are added to the cocktail. You gain an unarmed attack in the brawling group of your choice between a jaws attack that deals @Damage[1d6[piercing]] or a claw attack that has the agile trait and deals @Damage[1d4[slashing]].</li>\n<li><strong>Double</strong> This cocktail is just stronger, with more ginger and more rum. You gain resistance 1 to physical damage. The resistance increases to 2 for a moderate fury cocktail or 5 for a greater fury cocktail.</li>\n<li><strong>Mournful</strong> A few flower petals add a powerful aroma to the drink. You gain resistance 1 to void damage, or resistance 1 to vitality damage if you have void healing. The resistance increases to 5 for a moderate fury cocktail or 10 for a greater fury cocktail.</li>\n<li><strong>Skeptical</strong> A splash of bitters gives the drink a more complex flavor. You gain a +1 item bonus to saves against magic. This bonus is +2 for a moderate fury cocktail or +3 for a greater fury cocktail.</li>\n<li><strong>Titanic</strong> Yuzu juice and powdered giant hair are added to this cocktail. If you're Medium or smaller, you gain the following effects: you become Large, are @UUID[Compendium.pf2e.conditionitems.Item.Clumsy]{Clumsy 1}, and increase your reach by 5 feet (or by 10 feet if you started out Tiny).</li>\n<li><strong>Wyrmhide</strong> Pomegranate juice and elemental reagents are added to the cocktail. You gain resistance 1 to acid, cold, electricity, fire, and poison damage. The resistance increases to 5 for a moderate fury cocktail or 10 for a greater fury cocktail.</li>\n</ul>\n<p><strong>Drawback</strong> You take a –1 penalty to AC and a –2 penalty to Reflex saves.</p>"
```

Comment:
Typically effects are linked at the end of the description. Also, {Effect: Fury Cocktail (Greater)} is unneeded as it's the same as the effect name, so it will be added at build time.